examples/js/bots/liquidity_bot.ts

```diff
@@ -0,0 +1,66 @@
+import { defaultClient } from "../client";
+import { ORDER_SIDE_ASK, ORDER_SIDE_BID, ORDER_TYPE_LIMIT, TestUser } from "../config";
+
+const base = "DIF";
+const quote = "BTC";
+const market = `${base}_${quote}`;
+const botUseFunds = 0.8; // Use 80% of funds
+const deviation = 1.002;
+const tiersAmount = 10;
+const fee = "0";
+const balanceCache = {};
+
+const tick = async (user_id: TestUser) => {
+  await defaultClient.connect();
+  const balance = await defaultClient.balanceQuery(user_id);
+
+  const Q_BAL = (+balance.get(quote).available + +balance.get(quote).frozen) * botUseFunds;
+  const B_BAL = (+balance.get(base).available + +balance.get(base).frozen) * botUseFunds;
+
+  if (balanceCache[quote] === Q_BAL && balanceCache[base] === B_BAL) {
+    console.log("not trades happend, skipping");
+    return;
+  }
+
+  balanceCache[quote] = Q_BAL;
+  balanceCache[base] = B_BAL;
+
+  await defaultClient.orderCancelAll(user_id, market);
+
+  const getPrice = (tier: number, side: number) => {
+    return (Q_BAL * Math.pow(deviation, side * (2 * tier + 1))) / B_BAL;
```

Comment:
Looks good to me. 

`return Q_BAL * Math.pow(deviation, side * (2 * tier + 1)) / B_BAL;` would also be fine.